### PR TITLE
Documentation and Setup for release

### PR DESCRIPTION
**Pull request by nicolas-camacho**
Adding necesary files and changes for the release of the 0.0.1

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

**Review by kmorope — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1 +1,178 @@
...
+
+### Authenticate Challenge Payload <a name="authenticate-challenge-payload"></a>
+
+The challenge payload for authenticating a user is a JSON with the schema:
+
+```json
+{"publicKey":{"challenge":"WUM...2Mw","timeout":300000,"rpId":"your_backend","allowCredentials":[],"userVerification":"preferred"}}
+```
+
+## Running Tests <a name="running-tests"></a>
```
Comment (yafuquen) [inline README.md:97]: I would move this section to the end. We do not expect developers to run this part often, instead the instructions to run the sample app will be more used

File: src/CredentialManager.js [hunk 2]
```diff
@@ -1,3 +1,19 @@
+/*
+ * Copyright © 2024 Twilio Inc.
```
Comment (yafuquen) [inline src/CredentialManager.js:2]: Is this included in all the files? Even the sample app?

Comment (nicolas-camacho) [reply src/CredentialManager.js:2]: Yes, this is included in all files, except for those that works as config files, like `rollup.config.js` or `tsconfig.json`, this should be included on those files? also not in sample-app due to was added in a parallel PR

File: README.md (hunk 1 above)
Comment (nicolas-camacho) [reply README.md]: Done!

File: src/CredentialManager.js (hunk 2 above)
Comment (nicolas-camacho) [reply src/CredentialManager.js:2]: Already added to the sample-app

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+- Any other branch contains code with new features or improvements that are waiting to be merged into the `dev` branch.
+
+## Code of Conduct
+
+Help us keep `twilio-verify-passkeys-web` open and inclusive. Please be kind to and considerate of other developers, as we all have the same goal: make `twilio-verify-passkeys-web` as good as it can be.
+By interacting with the project in any shape or form you are agreeing to the project's [Code of Conduct](https://github.com/twilio/twilio-verify-passkeys-web?tab=coc-ov-file). If you feel like another individual has violated the code of conduct, please raise a complaint to [open-source@twilio.com](mailto:open-source@twilio.com).
+
+## Question or Problem?
+
+If you have questions about how to use `twilio-verify-passkeys`, please see our [docs](../README.md), and if you don't find the answer there, please contact [Twilio Support](https://www.twilio.com/help/contact) with any issues you have.
```
Comment (SergioFierro) [inline .github/CONTRIBUTING.md:28]: Shouldn't it be `twilio-verify-passkeys-web`?

File: LICENSE [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright 2020 Twilio, inc.
```
Comment (SergioFierro) [inline LICENSE:189]: I think it should be 2024

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (nicolas-camacho) [reply .github/CONTRIBUTING.md]: Fixed

File: LICENSE (hunk 4 above)
Comment (nicolas-camacho) [reply LICENSE]: Fixed

**Review by SergioFierro — APPROVED** [review]

**Review by kmorope — APPROVED** [review]

**Review by yafuquen — APPROVED** [review]

**Pull request merged**